docs/source/conf.py

```diff
@@ -50,13 +50,11 @@
     "nested_sections": True,
     # Directory where function/class granular galleries are stored
     "backreferences_dir": "_autosummary/backreferences",
-    # Modules for which function/class level galleries are created. In
-    # this case sphinx_gallery and numpy in a tuple of strings.
+    # Modules for which function/class level galleries are created
     "doc_module": ("synthesizer"),
-    #     # Regexes to match objects to exclude from implicit backreferences.
-    #     # The default option is an empty set, i.e. exclude nothing.
-    #     # To exclude everything, use: '.*'
-    #     # 'exclude_implicit_doc': {r'pyplot\.show'},
+    "abort_on_example_error": True,
+    # "show_memory": True,
```

Comment:
Any reason to leave these here commented?

Reply:
If show memory shows the memory usage in the logs that sounds useful!

Reply:
I thought about it but it's expensive. I think okay to leave here for users to comment out when testing locally?